modules/tensor_mechanics/include/kernels/ADStressDivergenceShell.h

```diff
@@ -0,0 +1,65 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ADKernel.h"
+#include "RankTwoTensor.h"
+#include "RankFourTensor.h"
```

Comment:
you could forward declare those like you do further below

Reply:
I have made this change